src/crm/apis/CustomObjectClassesApi.ts

```diff
@@ -56,890 +30,35 @@ import {
     MergeMetaRequest
 } from '../../merge_meta_request';
 
-export interface CustomObjectClassesAssociationTypesCreateRequest {
```

Comment:
This feels like a bad change mind bringing this file back?

Reply:
restored the file!